aot/src/main/java/com/dylibso/chicory/aot/AotMachine.java

```diff
@@ -367,13 +367,13 @@ private byte[] compileClass(String className, FunctionSection functions) {
         var classWriter = new ClassWriter(ClassWriter.COMPUTE_FRAMES);
         classWriter.visit(
                 Opcodes.V11,
-                Opcodes.ACC_PUBLIC | Opcodes.ACC_FINAL,
+                Opcodes.ACC_PUBLIC | Opcodes.ACC_FINAL | Opcodes.ACC_SUPER,
                 internalClassName,
                 null,
                 getInternalName(Object.class),
                 new String[] {getInternalName(Machine.class)});
 
-        classWriter.visitSource("wasm", "wasm");
+        classWriter.visitSource(null, null);
```

Comment:
Better to leave those fields empty than using dummy values.

Reply:
The second argument seems wrong for sure. The first one causes stack traces to include `wasm`, which seems useful, but I don't know if there are downsides. If @forax says we should remove it, then let's remove it!

Reply:
As electrum said, for me, it should be
```java
classWriter.visitSource("wasm", null);
```
so you get "wasm" printed in the stacktraces.

Reply:
Ok, I have nothing against, my reasoning was:
"better to not provide anything than a wrong `fileName` that the IDE will try to resolve".

But the IDE is still broken in either cases, so I just opened #576 to track it as a separate issue.

Thanks everyone for the feedback, much appreciated!